### ISSUE TITLE
Can I control the keyboardType for Zip code input?

### ISSUE THREAD

**Issue by xiao555** [stripe/stripe-react-native#1165]
When input zip code, the keyboard changed from numeric to default, which is not very user friendly.

<img width="200" src="https://user-images.githubusercontent.com/8833639/195003693-c1a86bb0-07db-49bd-8706-465e87f96502.png" /><img width="200" src="https://user-images.githubusercontent.com/8833639/195003747-a6221e9e-a4f1-430a-b733-d6fa430b613b.png" />

**Describe the solution you'd like**
Let developer can set the keyboardType for Zip code input.

**Describe alternatives you've considered**

**Additional context**

**Comment by sebastienlabine** [comment]
@xiao555 I think it really depends on the format of zip codes, countries like canada actually do need the regular keyboard because it contains alphanumeric characters

**Comment by xiao555** [comment]
@sebastienlabine Yes, but if only need numbers for zip code, maybe can do a optimization? Like add a prop we can use to set the keyboard type for zip code.

**Comment by xiao555** [comment]
@sebastienlabine Yes, but if only need numbers for zip code, maybe can do a optimization? Like add a prop we can use to set the keyboard type for zip code.

**Comment by manishnaruka** [comment]
There should a "Done" button to hide keyboard in numeric mode

**Comment by wooj-stripe** [comment]
@xiao555  Thank you for the suggestion! At this time, we are not planning on adding any new features to the Card Element. We are encouraging all users to instead use the Mobile Payment Element, which we launched in 2021. You can find the docs [here](https://docs.stripe.com/payments/accept-a-payment). If there are any reasons the MPE doesn't support your use case, we'd love to hear your feedback!
